video_xblock/video_xblock.py

```diff
@@ -385,12 +385,15 @@ def studio_view(self, _context):
             'players': PlayerName,
             'sources': TranscriptSource.to_dict().items(),
             # transcripts context:
-            'transcripts': transcripts,
+            'transcripts': filter_transcripts_by_source(
+                transcripts, sources=[TranscriptSource.THREE_PLAY_MEDIA], exclude=True
```

Comment:
Do you mean is there any occurrences where we use a full set of transcripts (w/o excluding 3PM ones)? I'm not sure I understand. Is this question about 'transcripts' field or about 'filter_transcripts_by_source' utility?